core/src/main/java/com/graphhopper/routing/AStarBidirection.java

```diff
@@ -1,14 +1,14 @@
 /*
  *  Licensed to GraphHopper GmbH under one or more contributor
- *  license agreements. See the NOTICE file distributed with this work for 
+ *  license agreements. See the NOTICE file distributed with this work for
  *  additional information regarding copyright ownership.
- * 
- *  GraphHopper GmbH licenses this file to you under the Apache License, 
- *  Version 2.0 (the "License"); you may not use this file except in 
+ *
```

Comment:
Looks like you [updated your IntelliJ version](https://discuss.graphhopper.com/t/whitespace-issue-in-graphhopper-with-intellij/2677) ? Maybe we can run the code formatting on the whole project to avoid having these diffs in every commit ?

Reply:
Yes, will do.

Reply:
When you are at it maybe make use of Java 7 diamond operator as well ('Explicit type can be replaced with <>' inspection in IntelliJ) :)